tests/store/filters/test_status_pool_filters.py

```diff
@@ -177,3 +181,66 @@ def test_filter_pools_without_invoice_id(
 
     # THEN pools should contain the test pool
     assert pools.all() and len(pools.all()) == 1
+
+
+def test_filter_pools_by_name_enquiry(
+    store_with_a_pool_with_and_without_attributes: Store,
+    name=StoreConftestFixture.NAME_POOL_WITH_ATTRIBUTES.value,
+):
+    """Test that a pool is returned when there is a pool with a specific name enquiry."""
+
+    # GIVEN a store with two pools of which one has a name enquiry
+
+    # WHEN getting pools with name enquiry
+    pools: Query = filter_pools_by_name_enquiry(
+        pools=store_with_a_pool_with_and_without_attributes._get_query(table=Pool),
+        name_enquiry=StoreConftestFixture.NAME_POOL_WITH_ATTRIBUTES.value,
+    )
+
+    # ASSERT that the query is a Query
+    assert isinstance(pools, Query)
+
+    # THEN pools should contain the test pool
+    assert pools.all() and len(pools.all()) == 1 and pools.all()[0].name == name
+
+
+def test_filter_pools_by_order_enquiry(
+    store_with_a_pool_with_and_without_attributes: Store,
+    name=StoreConftestFixture.NAME_POOL_WITH_ATTRIBUTES.value,
+):
+    """Test that a pool is returned when there is a pool with a specific order enquiry."""
+
+    # GIVEN a store with two pools of which one has an order enquiry
+
+    # WHEN getting pools with order enquiry
+    pools: Query = filter_pools_by_order_enquiry(
+        pools=store_with_a_pool_with_and_without_attributes._get_query(table=Pool),
+        order_enquiry=StoreConftestFixture.ORDER_POOL_WITH_ATTRIBUTES.value,
+    )
+
+    # ASSERT that the query is a Query
+    assert isinstance(pools, Query)
+
+    # THEN pools should contain the test pool
+    assert pools.all() and len(pools.all()) == 1 and pools.all()[0].name == name
+
+
+def test_filter_pools_by_customer_id(
+    store_with_a_pool_with_and_without_attributes: Store,
+    name=StoreConftestFixture.NAME_POOL_WITH_ATTRIBUTES.value,
+):
+    """Test that a pool is returned when there is a pool with a specific customer id."""
+
+    # GIVEN a store with two pools of which one has a customer id
+
+    # WHEN getting pools with customer id
+    pools: Query = filter_pools_by_customer_id(
+        pools=store_with_a_pool_with_and_without_attributes._get_query(table=Pool),
+        customer_ids=[1],
+    )
+
+    # ASSERT that the query is a Query
+    assert isinstance(pools, Query)
+
+    # THEN pools should contain the test pool
+    assert pools.all() and len(pools.all()) == 2 and pools.all()[0].name == name
```

Comment:
I think this makes it easier to determine the statement that failed
```suggestion
    assert pools.all()
    assert len(pools.all()) == 2
    assert pools.all()[0].name == name
```